darts/timeseries.py

```diff
@@ -2270,3 +2435,97 @@ def _set_freq_in_xa(xa_: xr.DataArray):
                 return TimeSeries(xa_)
 
         raise_log(IndexError("The type of your index was not matched."), logger)
+
+
+def concatenate(timeserie_sequence: Sequence['TimeSeries'],
+                axis: Optional[Union[str, int]] = 0,
+                ignore_time_axes: Optional[bool] = False):
```

Comment:
```suggestion
                axis: Union[str, int] = 0,
                ignore_time_axes: bool = False):
```

Reply:
I don't understand why have you removed `Optional`? Since we have defaults, this indeed is optional. Unless I'm not seeing something.

Reply:
Optional means it can either be `None` or a `bool`. It means the same as `Union[bool, None] = False`